python/dgl/function/message.py

```diff
@@ -168,15 +168,17 @@ def copy_e(e, out):
 
 def _gen_message_builtin(lhs, rhs, binary_op):
     name = "{}_{}_{}".format(lhs, binary_op, rhs)
-    docstring = """Builtin message function that computes message by performing
-    binary operation {} between {} feature and {} feature.
+    docstring = """Builtin message function that computes a message on an edge
+    by performing element-wise {} between features of {} and {}
+    if the features have the same shape; otherwise, it first broadcasts the features
```

Comment:
I think we elaborate more on broadcasting behaviour: https://github.com/dmlc/dgl/blob/master/python/dgl/kernel.py#L75-L85

Reply:
any reference to numpy broadcasting semantics?

Reply:
https://docs.scipy.org/doc/numpy/user/basics.broadcasting.html#general-broadcasting-rules